### DISCUSSION TITLE
How to add or be able to sell claim areas in the shopgui+ plugin

### DISCUSSION THREAD

**Discussion by ryerdevs** [GriefPrevention/GriefPrevention#2168] [Q&A]
Basically I would like my players to be able to purchase claim areas through the server store instead of using commands but I have no idea if it can be achieved although I have seen it on other servers

**Comment by romanshak** [comment]
You can do that using the sellclaimblocks / buyclaimblocks command and intergrate it witha gui by usuing placeholders.
